### ISSUE TITLE
False positive in MQTT "network is down" logic

### ISSUE THREAD

**Issue by johnvey** [andrewjfreyer/monitor#223]
**TO REPRO:**

This is on commit e22f7a9d

The logic for detecting when the network to the MQTT server is issuing false positives, resulting in the following log output:
```
[CMD-MQTT]   warning: network is down. enqueuing command to try again after a delay
[CMD-ERRO]   fatal mqtt error - messages may not be delivered as intended (network is down. enqueuing command to try again after a delay / 4)
```
The false positive is triggered if there are any devices in range that have a manufacturer's name that includes the string `network` (case-insensitive). For example, here are some registered manufacturer names that do:
```
Clover Network, Inc.
Baidu Online Network Technology (Beijing) Co., Ltd
Ayla Networks
Samsara Networks, Inc
Mobiquity Networks Inc
Swirl Networks, Inc.
```
(Source: [Bluetooth SIG member list](https://www.bluetooth.com/specifications/assigned-numbers/16-bit-uuids-for-members/?_ga=2.105995968.2115520943.1514979233-967253953.1449700706))

The fault lies in the error handler: https://github.com/andrewjfreyer/monitor/blob/585ef68f131fea370c73b0f18aee79fd4ced581f/support/mqtt#L167
Using a substring match for `network` is too simplistic here. It should be a more robust check on the entire message/format/return code.

**EXPECTED:**

The monitor should not log an error if there are devices that self-report names that collide with `network` (and likely `refused`).

**Comment by andrewjfreyer** [comment]
Excellent report. I can incorporate a fix easily.

**Comment by andrewjfreyer** [comment]
Fixed in 0.2.198
